integrations/microstream/metrics/src/main/java/io/helidon/integrations/microstream/metrics/MicrostreamMetricsSupport.java

```diff
@@ -0,0 +1,180 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.integrations.microstream.metrics;
+
+import java.util.Objects;
+
+import io.helidon.config.Config;
+import io.helidon.metrics.MetricsSupport;
+import io.helidon.metrics.RegistryFactory;
+
+import one.microstream.storage.embedded.types.EmbeddedStorageManager;
+import org.eclipse.microprofile.metrics.Gauge;
+import org.eclipse.microprofile.metrics.Metadata;
+import org.eclipse.microprofile.metrics.Metric;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.eclipse.microprofile.metrics.MetricType;
+import org.eclipse.microprofile.metrics.MetricUnits;
+import org.eclipse.microprofile.metrics.Tag;
+
+
+/**
+ *
+ * Helper class that provides the default metrics for an Microstream EmbeddedStorageManager.
+ *
+ */
+public class MicrostreamMetricsSupport {
+
+    private static final String CONFIG_METRIC_ENABLED_VENDOR = "vendor.";
+    static final String BASE_ENABLED_KEY = CONFIG_METRIC_ENABLED_VENDOR + "enabled";
+
+    private static final Metadata GLOBAL_FILE_COUNT = Metadata.builder()
+            .withName("microstream.globalFileCount")
+            .withDisplayName("total storage file count")
+            .withDescription("Displays the number of storage files.")
+            .withType(MetricType.GAUGE)
+            .withUnit(MetricUnits.NONE)
+            .build();
+
+    private static final Metadata LIVE_DATA_LENGTH = Metadata.builder()
+            .withName("microstream.liveDataLength")
+            .withDisplayName("live data length")
+            .withDescription("Displays live data length. This is the 'real' size of the stored data.")
+            .withType(MetricType.GAUGE)
+            .withUnit(MetricUnits.BYTES)
+            .build();
+
+    private static final Metadata TOTAL_DATA_LENGTH = Metadata.builder()
+            .withName("microstream.totalDataLength")
+            .withDisplayName("total data length")
+            .withDescription("Displays total data length. This is the accumulated size of all storage data files.")
+            .withType(MetricType.GAUGE)
+            .withUnit(MetricUnits.BYTES)
+            .build();
+
+    /**
+     * Create a new builder to construct an instance.
+     *
+     * @param embeddedStorageManager EmbeddedStorageManager instance that supplies the metrics data.
+     * @return A new builder instance
+     */
+    public static Builder builder(EmbeddedStorageManager embeddedStorageManager) {
+        return new Builder(embeddedStorageManager);
+    }
+
+    private Config config;

```

Comment:
Fields are always before methods.
Order within files in Helidon:
fields
constructor(s)
methods
classes

order of visibility:
public
package
protected
private

So in this case, all fields and the constructor are before static method `builder`

This comment is valid for all files in this PR. 
It would be great if you could do the reformatting yourself, so it is under your contribution. Another option would be that I do the reformatting, but that would make me author of a lot of changes in this code.

Reply:
fixed